core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -375,9 +376,13 @@ public boolean hasLostStorage() {
 
   @Override
   public String toString() {
+    int blockSizeLimit = 100;
```

Comment:
I feel like 100 is pretty low for this?

Reply:
This is a good question. I've no idea what is a good split and don't have statistics for the decision. I can only say a MasterWorkerInfo of a good 400 blocks can fill my 15-inch screen. 

The benefit of not truncating this list is we can search for a specific block id and know for sure if it exists on one worker. Once we start truncating there's no such nice certainty. So if we do truncate, intuitively I lean towards using a smaller limit, as we lose the certainty whatsoever.

Reply:
Can we do it like this, truncate in normal mode, and leave it as is if debug enabled.
@gpang WDYT about this in general?

Reply:
Good idea!